### DISCUSSION TITLE
Can't activate the Developer license

### DISCUSSION THREAD

**Discussion by fcicchi** [iTwin/itwinjs-core#5955] [General]
Dear team, 
I tried to contact you to activate the Developer tier and start using it for my company. 

I filled and sent the contact form twice in the past three weeks, but I didn't get any reply. 

Could you please let me know which is the best channel and contact to activate the tier?
My e-mail is: federico.cicchi@xyzreality.com

Kind Regards,
Federico Cicchi

**Comment by pratikshan85** [comment]
@fcicchi Can you please email us all the details to the following email? We will take care rest of the things.

tamanud.das@bentley.com
pratikshan.ghosh@bentley.com
arnab.gogoi@bentley.com

**Reply by fcicchi** [reply]
I'll do now, thank you!
